exporters/trace/zipkin/zipkin.go

```diff
@@ -0,0 +1,85 @@
+package zipkin
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"log"
+	"net/http"
+
+	export "go.opentelemetry.io/otel/sdk/export/trace"
+)
+
+type Exporter struct {
+	url    string
+	client *http.Client
+	logger *log.Logger
+}
+
+var (
+	_ export.SpanBatcher = &Exporter{}
+)
+
+type Options struct {
+	Client *http.Client
+	Logger *log.Logger
+}
+
+type Option func(*Options)
+
+func WithLogger(logger *log.Logger) Option {
+	return func(opts *Options) {
+		opts.Logger = logger
+	}
+}
+
+func WithClient(client *http.Client) Option {
+	return func(opts *Options) {
+		opts.Client = client
+	}
+}
+
+func NewExporter(url string, os ...Option) (*Exporter, error) {
+	opts := Options{}
+	for _, o := range os {
+		o(&opts)
+	}
+	if opts.Client == nil {
+		opts.Client = http.DefaultClient
+	}
+	return &Exporter{
+		url:    url,
+		client: opts.Client,
+		logger: opts.Logger,
+	}, nil
+}
+
+func (e *Exporter) ExportSpans(ctx context.Context, batch []*export.SpanData) {
+	if len(batch) == 0 {
+		// nothing to do
+		return
+	}
+	models := toZipkinSpanModels(batch)
+	body, err := json.Marshal(models)
+	if err != nil {
+		e.logf("failed to serialize zipkin models to JSON: %v", err)
+		return
+	}
+	e.logf("about to send a POST request to %s with body %s", e.url, body)
```

Comment:
This seems a bit verbose for normal operations.

Reply:
This is a no op by default, unless you configure the logger. And maybe it's lame, but I use that in tests too…

Reply:
Ah, didn't connect it was a no-op. SGTM